cookbook/controller/error_pages.rst

```diff
@@ -251,7 +251,25 @@ will be passed two parameters:
 Instead of creating a new exception controller from scratch you can, of course,
 also extend the default :class:`Symfony\\Bundle\\TwigBundle\\Controller\\ExceptionController`.
 In that case, you might want to override one or both of the ``showAction()`` and
-``findTemplate()`` methods. The latter one locates the template to be used.
+``findTemplate()`` methods. The latter one locates the template to be used. 
+
+.. note::
+  
+  In case of extending the :class:`Symfony\\Bundle\\TwigBundle\\Controller\\ExceptionController` 
+  you may configure a service to pass the twig-Environment and the debug-Flag to the constructor. 
+  
+  .. configuration-block::
+  
+      .. code-block:: yaml
+  
+          # app/config/config.yml
+          services:
+            my_custom_exception_controller_service:
```

Comment:
In modern Symfony apps, we recommend to use shorter service names. What about replacing `my_custom_exception_controller_service` by `app.exception_controller` ?